### DISCUSSION TITLE
我的博客 | Wendell

### DISCUSSION THREAD

**Discussion by giscus** [XiaoWendell/XiaoWendell.github.io#61] [Announcements]
# 我的博客 | Wendell

喧嚣中的静谧

https://wendell.website/posts/%E6%88%91%E7%9A%84%E5%8D%9A%E5%AE%A2/

**Comment by yen-chiu** [comment]
古有**孟母三迁**，今有我**博客三迁**。₍˄·͈༝·͈˄*₎◞ ̑̑

**Comment by yen-chiu** [comment]
又把评论系统更新了一下
